src/DynamicEvaluator/Application/Checker/BrokenPhpFilesChecker/Baseline/BaselineStorage.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace DynamicEvaluator\Application\Checker\BrokenPhpFilesChecker\Baseline;
+
+use DynamicEvaluator\Application\Checker\BrokenPhpFilesChecker\Dto\FileErrorDto;
+
+class BaselineStorage implements BaselineStorageInterface
+{
+    /**
+     * @var array<string, array<string>>
+     */
+    protected array $fileErrors;
+
+    /**
+     * @param \DynamicEvaluator\Application\Checker\BrokenPhpFilesChecker\Dto\FileErrorDto $fileErrorDto
+     *
+     * @return void
+     */
+    public function addFileError(FileErrorDto $fileErrorDto): void
+    {
+        if ($this->hasFileError($fileErrorDto)) {
```

Comment:
it's a strange method. You shouldn't check that this DTO is empty. you should prevent to create empty DTO

Reply:
It doesn't check check dto for empty status